### PR TITLE
Place the main frame in the middle of the screen when offscreen at startup (fix #228)

### LINKED ISSUE #228: Window location incorrect for multiple monitors
When Museeks was last used on a second monitor, it reopens in the same position.
For me, this is {-1312, 90}.
If I now only have one monitor connected Museeks is off the screen to the left.
The only way to get it back is to reconnect the monitor or edit the config file.
If both monitors were active I could use Win+Shift+Right to move it to my current one but this doesn't work when only one is active.
When switching from two screens to one when Museeks is already open, it successfully moves to the active one.

Only noticed on Windows 10, Museeks 0.7.1, can possibly try linux if desperate.

### PR DESCRIPTION
**Pull request by KeitIG**

### CODE REVIEW

**Comment by YurySolovyov** [comment]
I think it is better to put things like `(fix #228)` to the body of the issue to make it actually linkable

File: src/main-process/main.js [hunk 1]
```diff
@@ -61,6 +61,21 @@ app.on('ready', () => {
...
+    // check if the browser window is offscreen
+    const display = electron.screen.getDisplayNearestPoint(bounds).workArea;
+
+    const onScreen = bounds.x >= display.x
+        && bounds.x + bounds.width <= display.x + display.width
+        && bounds.y >= display.y
+        && bounds.y + bounds.height <= display.y + display.height;
+
+    if(!onScreen) {
+        delete bounds.x;
```
Comment (KeitIG) [inline src/main-process/main.js]: Do you mean "overwrite with the x and y bounds" or "overwrite the whole `bounds` object" ?

Comment (YurySolovyov) [inline src/main-process/main.js]: > overwrite with the x and y bounds

Comment (KeitIG) [inline src/main-process/main.js]: Well, I'll set them to `null` so, because Electron will auto-center the window then

Comment (YurySolovyov) [inline src/main-process/main.js]: can you at least put it into separate method?

Comment (KeitIG) [inline src/main-process/main.js]: Done. Cleanest way I found to do that for now.

**Pull request merged**